src/ProgressBar.js

```diff
@@ -58,7 +52,14 @@ const propTypes = {
   label: PropTypes.node,
   srOnly: PropTypes.bool,
   striped: PropTypes.bool,
-  active: PropTypes.bool,
+  animated: PropTypes.bool,
+  bsPrefix: PropTypes.string,
+  /**
+   * Sets the background class of the progress bar.
+   *
+   * @type ('success'|'danger'|'warning'|'info')
+   */
+  variant: PropTypes.string,
```

Comment:
what about making this prop type definition exact as the comment?
```js
PropTypes.oneOf(['success','danger','warning','info'])
```

Reply:
we want to allow folks to extend with variants of their own with minimal effort so the string is a bit more fexible